templates/blank.tmpl

```diff
@@ -1 +1 @@
-{{.Content}}
+{{.contents}}
```

Comment:
I think we still need the `header` and `footer` templates so that the email will be properly formatted with css.
```suggestion
{{ template "header" . }}
{{.contents}}
{{ template "footer" . }}
```

Reply:
I did find at least 1 other place in the terrain service that is using this blank email template: when a user is added as a Collection admin, they get a notification email with a subject like "Added as collection admin to ...", but that service sends a `null` contents message body.

This sends an email with only a subject line, and an empty body. If we add the header and footer to this blank template now, then I wonder if an email body containing only the header and footer might be confusing (or a little bothersome once you look over the boilerplate of the fancy email only to realize there was no actual message). 🤔

Reply:
Maybe we should allow the blank template to be used with an empty body, or very short messages without the header and footer, and I can open another PR to add a new template for DOIs, that will use the formatted email with the header and footer?

Reply:
Sounds good to me. Also found that `permanent_id_request` is missing the `header` template. Is that `template` is used? If so that needs to be fixed as well?

Reply:
Yes, that template is used to send notifications to DOI Request admins for new requests, and it looks like that also has an import error in the template. It's now using `user` instead of `username`: https://github.com/cyverse-de/iplant-email/blob/c6417d552b6937212ec0f5cc8e0c759ee810624d/conf/permanent_id_request.st#L2

I can also fix this template (and add the header) in the new PR.